samples/tls_between_enclaves/AttestedTLSREADME.md

```diff
@@ -0,0 +1,153 @@
+# Secure Channel in OE SDK
+
+  As Open Enclave SDK getting adopted into more realistic scenarios, we are receiving requests from OE SDK developers for adding secure channel support.
+
+  We do have the remote_attestation sample that shows how to conduct remote attestation between two enclaves and establish a `proprietary` channel based on asymmetric keys exchanged during the attestation process. It demonstrates how to conduct mutual attestation but it does not go all the way to show how to establish a fully secure channel.
+
+  Most of the real world software uses TLS-like standard protocol through popular TLS APIs (OpenSSL, WolfSSL, Mbedtls...) for establishing secure channels. Thus, instead of inventing a new communication protocol, we implemented `Attested TLS` feature to address above customer need by adding a set of new OE SDK APIs to help seamlessly integrate remote attestation into the popular TLS protocol for establishing an TLS channel with attested connecting party without modifying existing TLS APIs (such as OpenSSL, Mbedtls, and others).
+
```

Comment:
You may want to reference the Attested TLS paper here.

Reply:
addressed